.github/workflows/pull-request-checks.yml

```diff
@@ -1,6 +1,9 @@
-name: Validate Code
+name: Build, Test, and Pack
 
-on: [push]
+on:
```

Comment:
I'm having serious difficulty finding the documentation on this `on` keyword, can you point me in the right direction?

Reply:
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#on